util/src/main/java/tech/pegasys/artemis/util/backing/BasicView.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.artemis.util.backing;
```

Comment:
(optional) Not sure how much of a headache it would be to move this package, but I think we should avoid dropping big libraries into util and instead try to make independent packages:
```suggestion
package tech.pegasys.artemis.backingtree;
```

Reply:
I'm thinking about merging `backing` and `SSZTypes` packages and then we can probably create a distinct sub-module for all the ssz related stuff. Let's consider this option for the subsequent refactor?